.travis.yml

```diff
@@ -28,4 +28,8 @@ after_script:
     # Set version to timestamp
     npm --no-git-tag-version version $($(npm bin)/json -f package.json version)-prerelease.$(date +%s)
     npm publish
+    # Publish to gh-pages as most recent committer
+    git config --global user.email $(git log --pretty=format:"%ae" -n1)
+    git config --global user.name $(git log --pretty=format:"%an" -n1)
+    npm run --silent deploy -- -x -r https://${GH_TOKEN}@github.com/${TRAVIS_REPO_SLUG}.git
```

Comment:
If there's any concern about these config values affecting other `git` commands in undesirable ways, then environment variables are an option too: https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables#Committing